pkg/operator/hive/hive_controller.go

```diff
@@ -155,26 +155,31 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	// Monitor changes to DaemonSets:
 	err = c.Watch(&source.Kind{Type: &appsv1.DaemonSet{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &hivev1.HiveConfig{},
+		OwnerType: &hivev1.HiveConfig{},
 	})
 	if err != nil {
 		return err
 	}
 
 	// Monitor changes to Deployments:
 	err = c.Watch(&source.Kind{Type: &appsv1.Deployment{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
-		OwnerType:    &hivev1.HiveConfig{},
+		OwnerType: &hivev1.HiveConfig{},
 	})
 	if err != nil {
 		return err
 	}
 
 	// Monitor changes to Services:
 	err = c.Watch(&source.Kind{Type: &corev1.Service{}}, &handler.EnqueueRequestForOwner{
-		IsController: true,
```

Comment:
These IsController fields were incorrect I take it?

Reply:
Yeah. The owner references are setup with `IsController: false`. I found that when I copied and pasted for my watch and the watch didn't fire.

I tested this change for each of the watch types except daemonsets as I couldn't find any daemonsets that had the hiveconfig as an owner.